modules/olm-fb-catalogs-schemas.adoc

```diff
@@ -117,9 +117,10 @@ It is valid for an entry's replaces value to reference another bundle name that
 ----
 ====
 
-////
-For more information about defining upgrade edges, see the upgrade graph reference documentation.
-////
+[WARNING]
+====
+When using the `skipRange` field, the skipped Operator versions are pruned from the update graph and are therefore no longer installable by users with the `spec.startingCSV` property of `Subscription` objects. If you want to have direct (one version hop) updates to an Operator version from multiple previous versions, but also keep those previous versions available to users for installation, always use the `skipRange` along with the `replaces` field pointing to the immediate previous version of the Operator version in question.
```

Comment:
Open to alternatives for "hop", but FWIW I did see a handful of other uses in the repo already, and no warning against it in IBM or RH style guides.

Reply:
Actually ended up changing "one version hop" to "one version increment", and hoping that doesn't change the meaning really.